great-idea-website/index.html

```diff
@@ -15,49 +15,71 @@
 </head>
 
   <body>
-
-  Services
-  Product
-  Vision
-  Features
-  About
-  Contact
-
-  <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
-  Innovation
-  On
-  Demand
-
-  Get Started
-
-  <img src="img/header-img.png" alt="Image of a code snippet.">
-
-  Features
-  Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
-
-
-  About
-  Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
-
-  <img class="middle-img" src="img/mid-page-accent.jpg" alt="Image of code snippets across the screen">
-  
-  Services
-  Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
-
-
-  Product
-  Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
-
-  Vision
-  Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
-
-  Contact
-  123 Way 456 Street
-  Somewhere, USA
-  1 (888) 888-8888
-  sales@greatidea.io
-
-  Copyright Great Idea! 2018
-  
+    <header>
+      <nav>
+          <a href="#">Services</a>
+          <a href="#">Product</a>
+          <a href="#">Vision</a>
+          <a href="#">Features</a>
+          <a href="#">About</a>
+          <a href="#">Contact</a>
+        <div>
+          <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
+        </div>
+        <!-- <img class="logo" src="img/logo.png" alt="Great Idea! Company logo."> -->
+      </nav>
+      <!-- <img class="logo" src="img/logo.png" alt="Great Idea! Company logo."> -->
+    </header>
+    <section id="top">
+      <div id="innovation">
+          <h1 >Innovation</h1>
+          <h2 class="in">On</h2>
+          <h1 >Demand</h1> <br>
+         <button>Get Started</button>
+      </div>
+      <div class="left" id="img-div">
+          <img src="img/header-img.png" alt="Image of a code snippet.">
+      </div>
+    </section>
+    <hr>
+    <section>
+      <div class="sec-section">
+          <div class="right">
+              <h1>Features</h1>
```

Comment:
It's not a good practice to use more then one `<h1>` in the page. Normally it should work as the "title" of the page. 
You could substitute all of them with `<h2>` or `<h3>` .
You can read more about this here: https://www.searchenginejournal.com/header-tags-seo-best-practices/261835/